docs/src/modules/utils/postProcessImport.ts

```diff
@@ -18,7 +18,7 @@ export const ADAPTER_TO_LIBRARY: Record<string, string> = {
   AdapterMomentJalaali: 'moment-jalaali',
 };
 
-const PICKERS_ADAPTER_REGEX = /^@mui\/(lab|x-date-pickers)\/(?<adapterName>Adapter.*)/;
+const PICKERS_ADAPTER_REGEX = /^@mui\/(lab|x-date-pickers(?:-pro)?)\/(?<adapterName>Adapter.*)/;
```

Comment:
We could go with the following if you think that it makes the code more readable. 🤔 
```suggestion
const PICKERS_ADAPTER_REGEX = /^@mui\/(lab|x-date-pickers|x-date-pickers-pro)\/(?<adapterName>Adapter.*)/;
```

Reply:
I'm honestly fine with both, it's not a regexp we need to tweak regularly

Reply:
Agree with @flaviendelangle ... both are fine